MAUI/DataGrid/sorting.md

```diff
@@ -244,6 +244,20 @@ public class CustomSortComparer : IComparer<object>, ISortDirection
 }
 {% endhighlight %}
 {% endtabs %}
+## Sort Icon Customization
+The Sort Icon Customization feature allows you to personalize the appearance of sorting icons in the application according to your preferences. You can customize the colors of the sorting icons to match your preferred color scheme or visual preferences, enhancing the overall look and feel of your user interface.

```

Comment:
## Change sort icon color

The default sort icon color can be customized by setting the `DataGridStyle.SortIconColor` property.